src/foraging_gui/Foraging.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, parent=None,box_number=1,start_bonsai_ide=True):
         self.ToInitializeVisual = 1 # Should we visualize performance
         self.FigureUpdateTooSlow = 0# if the FigureUpdateTooSlow is true, using different process to update figures
         self.ANewTrial = 1          # permission to start a new trial
-        self.to_save = 1            # permission to save backup data
+        self.save_backup_allowed = 1   # permission to save backup data; 0, the previous saving has not finished, and it will not trigger the next saving; 1, it is allowed to save backup data
```

Comment:
In this case, I think `previous_backup_completed` sounds even clearer.

Reply:
Sounds good! Updated!